opennms-doc/guide-admin/src/asciidoc/text/minion/offheap.doc

```diff
@@ -0,0 +1,26 @@
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+=== Using Offheap storage for Sink Messages
+
+If minion experiences connectivity issues with _Kafka_ or _ActiveMQ_ there is a chance that Traps and Syslog messages may be lost as the internal queue is limited by heap size.
+Offheap option will extend the queues outside heap to facilitate more storage for the sink messages.
+
+==== Configuring offheap storage
+
+Install offheap feature on minion:
+
+[source, sh]
+----
+echo 'opennms-core-ipc-sink-offheap' >> "$MINION_HOME/etc/featuresBoot.d/features.boot"
+----
+
+Configure storage and enable offheap as below:
+
+[source, sh]
+----
+echo 'offHeapSize=1000000000
```

Comment:
Can we enhance this to support units `MB` and `GB` units? i.e. a user should be able to set the size to 2GB without having to do the math for the byte conversion.